app/assets/src/components/Samples.jsx

```diff
@@ -1870,12 +1873,16 @@ function SampleCardCheckboxes({
             key={`sample_${sample.db_sample.id}`}
```

Comment:
Can you remove the `onClick` and `data-sample-id` props? They aren't needed, and it makes this code more confusing.